docs/build/guides/nft.md

```diff
@@ -153,7 +130,7 @@ access(all) contract FooBar {
 
     init() {
         self.totalSupply = 0
```

Comment:
This field is removed above, not sure if it should be here or the intention was to remove it